src/config/dynamic-theme-fixes.config

```diff
@@ -13694,14 +13696,27 @@ img[alt="People using Notion"]
 .api-section-image .next-image
 .visual .next-image
 .logo svg > :first-child
+img[alt="Books on a shelf"]
+img[alt ="Some building blocks"]
+img[alt ="A team of people collaborating"]
+.logo-limit-height
+.notranslate::after
 
 CSS
 body {
-    background-color: var(--darkreader-neutral-background) !important;
+background-color: var(--darkreader-neutral-background) !important;
 }
 .notion-divider-block div div {
-    border-bottom: 1px solid ${rgba(55, 53, 47, 0.4)} !important;
-}
+border-bottom: 1px solid ${rgba(55, 53, 47, 0.4)} !important;}
+[data-darkreader-inline-fill]
+{fill: #ffffff !important;}
+.header.jsx-2604928324 {background-color: var(--darkreader-neutral-background) !important;}
+.person.jsx-2691780686{
+background-color: var(--darkreader-neutral-background) !important;}
+.notranslate {  color: var(--darkreader-neutral-text) !important;}
+.notion-selectable notion-callout-block{background-color: var(--darkreader-neutral-background) !important;}
+#notion-app > div > div.notion-overlay-container.notion-default-overlay-container > div:nth-child(2) > div > div:nth-child(2) > div > div:nth-child(1) > div.notion-scroller.vertical > div.whenContentEditable > div:nth-child(4) > div > div > div.notion-selectable.notion-callout-block > div > div
+{background-color: var(--darkreader-neutral-background) !important;}
```

Comment:
Please format these, we don't one-liners CSS in this file.

Reply:
I'm going to create another pull request cause I changed a lot of the code, I also removed some of the redundant old code that was made by someone else and made it more efficient